ontology/yaml/resources/entity_types/global.yaml

```diff
@@ -12,6 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+EQUIPMENT:
+  id: "9693662434551660544"
+  description: "A piece of equipment."
+  is_abstract: true
+  opt_uses:
+  - manufacturer_label
+  - model_label
+
+MTS:
```

Comment:
Do you mean equipment? What is this, and how does it differ from equipment?

Reply:
MTS is a generic multisensor device , that can be used for any discipline (HVAC, lighting, etc.). It can have any type of sensor, like [this one](https://www.beringar.co.uk/product) which even has a people counting camera besides CO2, temperature, humidity, etc.

Reply:
I don't know that we want to be this vague. Equipment can be multi-sensor devices also. Anything other than a stand-alone sensor is a MTS. I think we want more specifics. The link you added is an occupancy sensor with some extra stuff; an Enlighted fixture is a light with some other sensors (even occ sensors). I think more specificity around purpose is necessary.